corehq/apps/integration/kyc/views.py

```diff
@@ -147,6 +149,16 @@ def verify_rows(self, request, *args, **kwargs):
 
         return self.render_htmx_partial_response(request, 'kyc/partials/kyc_verify_alert.html', context)
 
+    def _filter_valid_users(self, kyc_users):
+        valid_users = []
+        kyc_user_api_fields = self.kyc_config.get_api_field_to_user_data_map_values().values()
+        for kyc_user in kyc_users:
+            is_user_invalid = any(not kyc_user.get(field) for field in kyc_user_api_fields)
+            if is_user_invalid:
+                continue
+            valid_users.append(kyc_user)
+        return valid_users
+
```

Comment:
Nit: A list comprehension could simplify that `for` loop.

```suggestion
    def _filter_valid_users(self, kyc_users):
        def is_user_valid(kyc_user):
            return all(kyc_user.get(field) for field in kyc_user_api_fields)

        kyc_user_api_fields = self.kyc_config.get_api_field_to_user_data_map_values().values()
        return [kyc_user for kyc_user in kyc_users if is_user_valid(kyc_user)]
```

Reply:
https://github.com/dimagi/commcare-hq/pull/36438/commits/d831938abb50acccadfeabe4d997f098d724d3cf